README.md

```diff
@@ -164,9 +163,117 @@ modelSelector | JSON | Use the Cloudant Query selector syntax to associate model
 
 ### _rev Property
 
-In a document, property `_rev` is the latest doc revision and must be provided when modifying the doc. Currently when updating/deleting a doc, the connector first retrieves that doc, then gets its `_rev`, then calls a modify api with `_rev` included. It makes three calls to the database and this would easily result in race condition.
-  
-Considering the nature of Cloudant, we plan to give user the flexibility to define `_rev` property in model and provide `_rev` by themselves instead of the connector. In such way user can get the revision and control conflict before using that revision to modify the document.
+In a document, property `_rev` is the latest doc revision and must be provided when modifying the doc.
+
+Our connector allows the user to retrieve back the `_rev` property upon all CRUD operations, however does not add it to the model definition.
+
```

Comment:
What extra line? If you see the line numbers on the side, we have `Our connector allows the user to retrieve back the `_rev` property upon all CRUD operations, however does not add it to the model definition.` on L168 and then space out  L169 empty and write the next sentence on L170.